app/client/src/styles/components/templates.styl

```diff
@@ -1,12 +1,7 @@
 @import '../variables/sizes'
 @import '../variables/colors'
 
-h1
-  font-size 0.8em
-  letter-spacing 0.2em
-  text-transform uppercase
-  color $templates-text-color
-  margin-bottom 20px
+
```

Comment:
One last tiny thing to fix, can you remove this newline? Thanks! :D

Reply:
Hardest problem yet! :P. 

I'll get around to it tomorrow. Is there a style guide or is this a .styl thing?

Reply:
So there is a style guide for the JS and the Stylus code. For the JS, I'm using a custom style guide which I call [blyss](https://github.com/saadq/blyss). It is basically using the [standard](github.com/feross/standard) style guide with some modifications.

For the Stylus, I'm using [stylint](https://github.com/SimenB/stylint) to test the styles. You can find the linting rules I'm using [here](https://github.com/saadq/latexresu.me/blob/master/app/client/.stylintrc).

So normally you would get an error in the console if you didn't follow the style guide, however I don't think I have any rule set for extra newlines which is why you didn't get any message.

Reply:
Alright, now there is just one newline under the @imports